lib/mumuki/domain/helpers/user.rb

```diff
@@ -63,6 +63,10 @@ def full_name
 
   alias_method :name, :full_name
 
+  def random_avatar
+    Avatar.order('RANDOM()').first
```

Comment:
This a curious problem that I've never dealt with.
I'm sure you've read the docs but just to make it explicit here:

This method is the fastest but it has the downside of not being portable nor between different DBs nor between Rails versions (doesn't work in Rails 6)

`Avatar.offset(rand(Avatar.count)).first` is the alternative but it's a bit more verbose and inefficient.

Thinking a bit ahead, we'll probably change this one throughout the development of the feature thinking that _maybe_ not all avatars are initially available.
I'd leave it this way with a comment for now. 

Having said that, I'd move this method to Avatar and call `Avatar.random` here.

Reply:
Or even better, `Avatar.sample`, which is consistent with collections naming: `[1, 2, 3, 4].sample` returns an element in a random position

Reply:
Definitely `sample` is more consistent!